mmv1/third_party/terraform/services/logging/resource_logging_sink.go

```diff
@@ -112,6 +112,7 @@ func expandResourceLoggingSink(d *schema.ResourceData, resourceType, resourceId
 		Filter:          d.Get("filter").(string),
 		Description:     d.Get("description").(string),
 		Disabled:        d.Get("disabled").(bool),
+		WriterIdentity:  d.Get("writer_identity").(string),
```

Comment:
`writer_identity` is an output-only field, was specifying it required for update?

Reply:
`writer_identity` is an output-only field. It's not required or recommended in a request. If a user provides it in a request, the provided value will be ignored, and actual value will be assigned accordingly.

Reply:
will remove this field.